src/renderers/WebGLRenderer.js

```diff
@@ -1196,7 +1200,7 @@ function WebGLRenderer( parameters ) {
 					var geometry = objects.update( object );
 					var material = object.material;
 
-					currentRenderList.push( object, geometry, material, _vector3.z, null );
+					currentRenderList.push( object, geometry, material, groupOrder, _vector3.z, null );
```

Comment:
You are passing in `groupOrder` and setting the last arg to `null` because it is not a group???

Reply:
The last argument is for multi-materials.

Reply:
I see. Group vs. groups.

Reply:
Yeah, sorry. Group vs material group.